src/Kentico.Xperience.Lucene/Models/LuceneIndex.cs

```diff
@@ -17,26 +55,34 @@ public sealed class LuceneIndex
     public Analyzer Analyzer { get; }
 
     /// <summary>
-    /// The type of the class which extends <see cref="LuceneSearchModel"/>.
+    /// The type of the class which extends <see cref="ILuceneIndexingStrategy"/>.
     /// </summary>
-    public Type LuceneSearchModelType
+    public ILuceneIndexingStrategy LuceneIndexingStrategy
     {
         get;
     }
 
+
     /// <summary>
-    /// The type of the class which extends <see cref="ILuceneIndexingStrategy"/>.
+    /// The code name of the Lucene index.
     /// </summary>
-    public ILuceneIndexingStrategy LuceneIndexingStrategy
+    public string IndexName
     {
         get;
     }
 
+    /// <summary>
+    /// The Name of the WebSiteChannel.
+    /// </summary>
+    public string WebSiteChannelName
+    {
+        get;
+    }
 
     /// <summary>
-    /// The code name of the Lucene index.
+    /// The Language used on the WebSite on the Channel which is indexed.
     /// </summary>
-    public string IndexName
+    public List<string> LanguageCodes
```

Comment:
```suggestion
    public List<string> LanguageNames
```

See previous comments